.github/workflows/github_actions_linter.yml

```diff
@@ -0,0 +1,23 @@
+name: GitHub Actions linter
```

Comment:
I'm not sure how useful this is given that it'll be used during initial set up... but happy to give it a go!

Reply:
This is simply a workflow that keeps an eye on the GitHub Actions workflows syntax. It will only run when a workflow changes.